docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
+
+
+Refresh control is a mechanism for controlling the revisions of a snap that will be offered as updates  to a device. A *gating snap* can control the revisions of a *gated snap* so that, if the *gating snap* is installed on a device, only revisions of the *gated snap* which have been validated by the publisher of the *gating snap* can be installed or refreshed to.
+
+A gating snap can control revisions of one or more gated snaps. Similarly, a gated snap can require verification from more than one gating snap, and a particular revision of the gated snap will only be installable or refreshable to, if it satisfies the constraints of all its gating snaps. However, because it can be confusing to device operators, use of multiple gating snaps to control one specific gated snap is not recommended. The best practice is to use a single gating snap for each gated snap.
```

Comment:
Consider highlight the fact that this is not recommended, perhaps like this. Note: While it is possible for a gated snap to require verification from multiple gating snaps, this is not recommended since it leads to complexity...."

Reply:
Thanks for the suggestions!

I am mentioning that it is not recommended, so it sounds like a matter of style to me. I would suggest merging as-is and then having someone (you could be that person!) go over style and apply any corrections; I won't object to any style changes (since I don't own canonical-docs, I wouldn't even be in the loop ;)